apps/web/src/components/healthcare/ClinicalWorkflowPatterns.tsx

```
@@ -0,0 +1,851 @@
+              <Calendar className="w-8 h-8 mx-auto mb-2 opacity-50" aria-hidden="true" />
+              <p>Nenhum horário disponível para este tipo de consulta</p>
+            </div>
+          )}
+        </div>
+
+        {/* Notes */}
+        {selectedSlot && (
+          <div className="space-y-2">
+            <label className="text-sm font-medium text-[#112031]">Observações (opcional)</label>
+            <Textarea
+              value={notes}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)